src/Illuminate/Support/Str.php

```diff
@@ -949,6 +949,19 @@ public static function slug($title, $separator = '-', $language = 'en')
         return trim($title, $separator);
     }
 
+    /**
+     * Join strings with a single space, ignoring nullable values.
+     *
+     * @param  mixed  ...$words
+     * @return string
+     */
+    public static function space(mixed ...$words): string
```

Comment:
Isn't it better to call this method `join`  and allow the user to pass the glue as a parameter?